src/components/Patient/SampleDetails.tsx

```diff
@@ -1,10 +1,13 @@
 import { camelCase, capitalize, startCase } from "lodash-es";
-import { navigate } from "raviger";
+import { Link, navigate } from "raviger";
 import { useTranslation } from "react-i18next";
 
 import Card from "@/CAREUI/display/Card";
 
-import ButtonV2 from "@/components/Common/ButtonV2";
+import { Badge } from "@/components/ui/badge";
+import { Button } from "@/components/ui/button";
+import { CardContent, CardHeader } from "@/components/ui/card";
```

Comment:
Let's not mix shadcn's card with our card. Switch to using shadcn's card.

Reply:
@rithviknishad shadcn Card added, old Card removed